doc/release-notes/4259-java-11.md

```diff
@@ -6,34 +6,41 @@ The Dataverse software now runs and requires Java 11. This will provide performa
 
 ## Notes to Admins
 
-### Java 11
+### Java 11 Upgrade
 
 There are some things to note and keep in mind regarding the move to Java 11:
 
-- You should install the JDK/JRE following your usual methods, depending on your operating system.
+- You should install the JDK/JRE following your usual methods, depending on your operating system. An example of this is:
+
+    `$ sudo yum remove java-1.8.0-openjdk java-1.8.0-openjdk-devel java-1.8.0-openjdk-headless`  
+
+    `$ sudo yum install java-11-openjdk-devel`  
+    
```

Comment:
should we specify for what OS / environment this example is for?

Reply:
updated in https://github.com/IQSS/dataverse/pull/7544/commits/bf393cc101832c471315f29284c8f103dc3028d6